src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/SolutionExplorer/PackageManagerUICommandHandler.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Threading;
+using Microsoft.VisualStudio.Workspace.VSIntegration.UI;
+using IAsyncServiceProvider = Microsoft.VisualStudio.Shell.IAsyncServiceProvider;
+using Task = System.Threading.Tasks.Task;
+
+namespace NuGet.VisualStudio.OnlineEnvironment.Client
+{
+    internal class PackageManagerUICommandHandler
+    {
+        private const bool IsPackageManagerUISupportAvailable = false;
+        private readonly JoinableTaskFactory _joinableTaskFactory;
+        private readonly IAsyncServiceProvider _asyncServiceProvider;
+
+        public PackageManagerUICommandHandler(JoinableTaskFactory joinableTaskFactory, IAsyncServiceProvider asyncServiceProvider)
```

Comment:
`PackageManagerUICommandHandler` class is `internal` but the constructor is `public`.

Reply:
Good question, but I searched and found some reasoning we may want to keep it public https://stackoverflow.com/a/16500613

Reply:
MSDN [doc](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers#class-and-struct-member-accessibility). Since this class is not implementing any interface or overrides virtual methods that are defined in a public base class so I think the constructor can be marked as `internal`

Reply:
In the interest of saving time for everyone, I'll change it to internal.